agent/acs/handler/attach_eni_handler.go

```diff
@@ -154,11 +157,11 @@ type ackTimeoutHandler struct {
 func (handler *ackTimeoutHandler) handle() {
 	eniAttachment, ok := handler.state.ENIByMac(handler.mac)
 	if !ok {
-		seelog.Warnf("Timed out waiting for ENI ack; ignoring unmanaged ENI attachment with MAC address:%s", handler.mac)
+		seelog.Warnf("Timed out waiting for ENI ack; ignoring unmanaged ENI attachment with MAC address: %s", handler.mac)
 		return
 	}
 	if !eniAttachment.IsSent() {
-		seelog.Info("Timed out waiting for ENI ack; removing ENI attachment record with MAC address: %s", handler.mac)
+		seelog.Infof("Timed out waiting for ENI ack; removing ENI attachment record with MAC address: %s", handler.mac)
 		handler.state.RemoveENIAttachment(handler.mac)
```

Comment:
If the eventhandler was unable to submit the state change for the attachment within the timeout, there's no need maintaining the attachment state locally anyway, which I think is fine.